lib/SessionManager.js

```diff
@@ -65,7 +65,7 @@ module.exports = (function(){
     }
 
     function getSessionDuration() {
-        return config.get('sessionDuration');
+        return config.get('endlistAppederTimerMillisec');
```

Comment:
since nobody is using this value execpt EndlistAppender, i think only end list appender.js should have this code